PySDM/backends/impl_numba/methods/collisions_methods.py

```diff
@@ -50,6 +50,84 @@ def coalesce(  # pylint: disable=too-many-arguments
 
 @numba.njit(**{**conf.JIT_FLAGS, **{"parallel": False}})
 def break_up(  # pylint: disable=too-many-arguments,unused-argument
+    i,
+    j,
+    k,
+    cid,
+    multiplicity,
+    gamma,
+    attributes,
+    n_fragment,
+    max_multiplicity,
+    min_volume,
+    breakup_rate,
+    breakup_rate_deficit,
+    warn_overflows,
+    volume,
+):  # pylint: disable=too-many-branches
+    # TODO #874: update the arguments for the calling function
+
+    # 1. find the max gamma that can be supported in 1 time step, add to rate,
+    #    and add remainder to the deficit (limits: max_xi, max_volume, min_volume)
```

Comment:
max_xi and max_volume are mentioned here, but not featured in the code, let's turn comments here into function names and function arguments for clarity?